app/views/content_change_requests/_request_details.html.erb

```diff
@@ -81,36 +88,37 @@
       class: "input-md-6 form-control",
       aria: {
         describedby: "subject-area-hint"
-      }
+      },
+      required: true
     %>
   </span>
 </div>
 
 <div class="form-group">
   <span class="form-label">
-    <%= f.label :url, "Which URLs are affected?" %>
+    <%= f.label :url do %>
+      Which URLs are affected? <abbr title="required">*</abbr>
+    <% end %>
   </span>
   <p class="help-block hint-block--md-6" id="url-hint">
     List URLs, one per line.
   </p>
   <span class="form-wrapper">
-    <%= f.text_area :url, placeholder: "https://www.gov.uk/", class: "input-md-6 form-control", rows: 4, cols: 50, aria: {
+    <%= f.text_area :url, placeholder: "https://www.gov.uk/", required: true, class: "input-md-6 form-control", rows: 4, cols: 50, aria: {
       describedby: "url-hint"
     } %>
   </span>
 </div>
 
 <div class="form-group">
   <span class="form-label">
-    <%= f.label :related_urls do %>
-      Are any Welsh URLs affected? <abbr title="required">*</abbr>
-    <% end %>
+    <%= f.label :related_urls, "Are any Welsh URLs affected?" %>
   </span>
   <p class="help-block hint-block--md-6" id="related-urls-hint">
     Include any Welsh translations. List URLs, one per line.
   </p>
   <span class="form-wrapper">
-    <%= f.text_area :related_urls, required: true, aria: { required: true }, class: "input-md-6 form-control", rows: 4, cols: 50, aria: {
+    <%= f.text_area :related_urls, aria: { required: true }, class: "input-md-6 form-control", rows: 4, cols: 50, aria: {
```

Comment:
If this isn't required, we should remove the aria:

```suggestion
    <%= f.text_area :related_urls, class: "input-md-6 form-control", rows: 4, cols: 50, aria: {
```

Also the bit we've just removed was probably conflicting with the `aria: { describedby: "related-urls-hint"` bit (i.e. you have two values for `aria`, I'm not sure how Rails deals with that).